src/platform/Tizen/MdnsImpl.cpp

```diff
@@ -0,0 +1,65 @@
+/*
+ *
+ *    Copyright (c) 2021 Project CHIP Authors
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+#include "lib/mdns/platform/Mdns.h"
+
+using namespace chip::Mdns;
+
+namespace {
+
+} // namespace
+
+namespace chip {
+namespace Mdns {
+
+CHIP_ERROR ChipMdnsInit(MdnsAsyncReturnCallback successCallback, MdnsAsyncReturnCallback errorCallback, void * context)
```

Comment:
Why is this  file provided if nothing is implemented? would it be not easier to  say 'chip_mdns = none' ?

Reply:
* We will use tizen mdns APIs to operation, so we include chip_mdns feature, and will upload the patchsets sequentially.

This PR is an initial patchset for Tizen OS port. And we only include the minimum codes for build - the skeleton of functions. If we include all implements for functions, the code review will be very hard. 
So we are going to separate the implementation and upload it.